src/compiler/proposals-and-stabilization.md

```diff
@@ -18,8 +18,10 @@ are suitable for each method of making a proposal - see below):
     other team members to raise any concerns.
   - Seconding can only be used to approve a MCP.
 - FCP
-  - A final comment period will require sign-off from a majority of the compiler team to approve
-    a proposal and then a ten day waiting period.
+  - A Final Comment Period is started by a T-compiler member, it's a tool to get concrete consensus
+    from the team.
```

Comment:
Remark: there's a funny thing where an "FCP" can mean:

1. A `compiler-fcp` team consensus vote, which is a **full-team Final Comment Period**.
2. A Major Change Proposal acceptance *Final Comment Period* window, but that only requires one "second" and no blocking objections.

Lol.

Reply:
uhhh 

Let's pretend you didn't see anything.